rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -21,25 +19,24 @@ pub struct Tun {
 
 impl Tun {
     pub fn new(config: &InterfaceConfig) -> Result<Self> {
+        const TUNNEL_UUID: &str = "e9245bc1-b8c1-44ca-ab1d-c6aad4f13b9c";
+        const TUNNEL_NAME: &str = "Firezone Tunnel";
+
         // The unsafe is here because we're loading a DLL from disk and it has arbitrary C code in it.
-        // As a defense, we could verify the hash before loading it. This would protect against accidental corruption, but not against attacks. (Because of TOCTOU)
+        // TODO: As a defense, we could verify the hash before loading it. This would protect against accidental corruption, but not against attacks. (Because of TOCTOU)
```

Comment:
Not a horrible idea, but seems like a rare edge case since we are only ever reading this file.

What we had in `0.7` was a crashloop detector that would prevent endless crashloops, instructing admins to contact support and investigate further to resolve.

(just an idea) We could do something similar in the clients, and ask them to re-install. That would take care of not only this but any file corruption issue that we can't explicitly handle ourselves.

Reply:
In the `wintun_install` module in this same PR I'm already checking the hash to decide whether we should unpack the DLL again (maybe it was updated, for example) or reuse the on-disk file without using up a disk write. I'll remove this comment as redundant.